plutus-core/untyped-plutus-core/test/Evaluation/Builtins/Bitwise.hs

```diff
@@ -0,0 +1,1157 @@
+-- editorconfig-checker-disable-file
+{-# LANGUAGE LambdaCase        #-}
+{-# LANGUAGE MultiWayIf        #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TypeApplications  #-}
+{-# OPTIONS_GHC -Werror #-}
+
+module Evaluation.Builtins.Bitwise (
```

Comment:
I'm happy to explain anything that's not clear, but in reality, most of this is working around various vagarities of Hedgehog and property testing in general. I'm certainly happy to explain these things, but it would help if I could know what _specifically_ isn't clear or self-explanatory. Property-based testing is a large topic, and Hedgehog in particular requires some specific workarounds, which aren't trivial to explain.

Reply:
My "outside view" is that it is extraordinarily unlikely that 1000 lines of Haskell doesn't need any explanations. I'll start a review of this file and list some questions, perhaps that will help point out what needs explaining.